umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif9.clj

```diff
@@ -19,6 +19,7 @@
    [cmr.umm-spec.versioning :as umm-spec-versioning]
    [cmr.umm.dif.date-util :refer [parse-dif-end-date]]))
 
+;; TODO
```

Comment:
We no longer support dif9, but we still have some collections in UAT and SIT that we have to delete. I am pretty sure we removed all DIF 9 records from production. Reading from DIF 9 to UMM should be good as is. We will have to make sure UMM-C to DIF 9 doesn't break, it if does then we need to fix the translations to what ever the dif 9 valid values are, we are not changing the dif 9 schema

Reply:
I test locally with .dif

curl --location 'https://cmr.sit.earthdata.nasa.gov/search/collections.dif?concept_id=C1200000001-JM_PROV2' --header 'Accept: application/dif+xml' --header 'Echo-Token: ••••••'

Is this how you test dif9?

If so, this came back with 200 when given new enum values